features/manage-position/Withdraw.tsx

```diff
@@ -123,6 +124,10 @@ const Deposit = () => {
     );
     const hasPendingWithdraw = pendingWithdraw === "Yes";
     const pendingWithdrawTimeRemaining = withdrawPassTime - Number(currentTime);
+    const progressBarPercent =
```

Comment:
What if `withdrawlLiveness` is 0 will this crash?

Reply:
it doesent crash it but I agree it should not calculate this if it is zero. Have updated PTAL.